tests/cli/run.sh

```diff
@@ -76,6 +76,14 @@ run yardstick label apply $(yardstick result list -r test --ids -t grype@v0.50.2
 
 assert_last_output_contains "label: TruePositive"
 
+ID_TO_REMOVE="$(yardstick label list | grep id: | cut -d ' ' -f 2 | head -n 1)"
+LABEL_FILE=".yardstick/labels/docker.io+anchore+test_images@sha256:10008791acbc5866de04108746a02a0c4029ce3a4400a9b3dad45d7f2245f9da/${ID_TO_REMOVE}.json"
+BAK_FILE="${LABEL_FILE}.bak"
```

Comment:
nit: it would be nice to test the `yardstick label add` in a way like `ID_TO_REMOVE=$(yardstick label add ...)` and remove that file instead of one that is checked in and needing to back it up. In that way the CLI test acts more like a real user.